app/src/main/res/layout/view_repository.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
Also I think you want wrap_content for a repository view. So it takes as much as it needs, and not the entire screen